connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoader.java

```diff
@@ -162,7 +162,7 @@ public ClassLoader connectorLoader(String connectorClassOrAlias) {
                           : connectorClassOrAlias;
         PluginClassLoader classLoader = pluginClassLoader(fullName);
         if (classLoader == null) {
-            log.error(
+            log.trace(
```

Comment:
I agree this should not be an error. Would debug be reasonable given the "Getting plugin class loader" message above?